android/WorkshopCompleted/app/src/main/java/com/mobiledgex/workshopskeleton/MainActivity.java

```diff
@@ -380,7 +381,7 @@ public boolean findCloudlet() throws ExecutionException, InterruptedException, P
         //Find FqdnPrefix from Port structure.
         String FqdnPrefix = "";
         List<distributed_match_engine.Appcommon.AppPort> ports = mClosestCloudlet.getPortsList();
-        String appPortFormat = "{Protocol: %d, FqdnPrefix: %s, Container Port: %d, External Port: %d, Public Path: '%s'}";
+        String appPortFormat = "{Protocol: %d, Container Port: %d, External Port: %d, Path Prefix: '%s'}";
         for (Appcommon.AppPort aPort : ports) {
             FqdnPrefix = aPort.getFqdnPrefix();
```

Comment:
Unused?

Reply:
Yes. Just showing how to get the values. (Possibly copied from EmptyMatchingEngineApp.)